[2025-09-10T00:02:14.124936] [INFO] [SYSTEM] Start new run (run_id=ac601686-4562-4f36-acc2-cfc967ccb1e6)
[2025-09-10T00:02:38.144737] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 3
[2025-09-10T00:02:54.376119] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Counts computed for kit build.
[2025-09-10T00:02:57.985215] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Base64 length bounds computed for pad selection.
[2025-09-10T00:03:17.128817] [INFO] [GENERAL] debug_utils.py:log_debug:238 - Using PAD size (bucketized): 128
[2025-09-10T00:03:24.199978] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T00:03:26.562399] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T00:03:28.154187] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T00:03:28.155187] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Argon2id parameters confirmed for kit.
[2025-09-10T00:03:28.156188] [WARNING] [CRYPTO] debug_utils.py:log_debug:238 - Combinatorial hardness below policy threshold.
[2025-09-10T00:03:31.862350] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Combinatorial hardness evaluation complete.
[2025-09-10T00:03:31.864349] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T00:03:32.137719] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T00:03:32.139719] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T00:03:32.206231] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T00:03:32.207231] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T00:03:32.724417] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T00:03:32.726420] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T00:03:33.249173] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T00:03:34.859673] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Using user-provided password='Rex' (raw).
[2025-09-10T00:03:34.860673] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Starting Argon2id KDF (Noble). pass='Rex', salt(b64)='gD776GEf5lyyaz3W1h71Bw=='
[2025-09-10T00:03:34.967187] [ERROR] [GENERAL] debug_utils.py:log_exception:286 - Error during encrypted kit preparation
